examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,298 @@
+# Compute non-linear morph mapping
+
+def compute_morph_map(img_m, img_s=None, niter_affine=(100, 100, 10),
+                      niter_sdr=(5, 5, 3)):
+    # get Static to world transform
+    img_s_grid2world = img_s.affine
+
+    # output Static as ndarray
+    img_s = img_s.dataobj[:, :, :]
+
+    # normalize values
+    img_s = img_s.astype('float') / img_s.max()
+
+    # get Moving to world transform
+    img_m_grid2world = img_m.affine
+
+    # output Moving as ndarray
+    img_m = img_m.dataobj[:, :, :]
+
+    # normalize values
+    img_m = img_m.astype('float') / img_m.max()
+
+    # compute center of mass
+    c_of_mass = transform_centers_of_mass(img_s, img_s_grid2world,
+                                          img_m, img_m_grid2world)
+
+    nbins = 32
+
+    # set up Affine Registration
+    affreg = AffineRegistration(metric=MutualInformationMetric(nbins, None),
+                                level_iters=list(niter_affine),
+                                sigmas=[3.0, 1.0, 0.0],
+                                factors=[4, 2, 1])
+
+    # translation
+    translation = affreg.optimize(img_s, img_m, TranslationTransform3D(), None,
+                                  img_s_grid2world, img_m_grid2world,
+                                  starting_affine=c_of_mass.affine)
+
+    # rigid body transform (translation + rotation)
+    rigid = affreg.optimize(img_s, img_m, RigidTransform3D(), None,
+                            img_s_grid2world, img_m_grid2world,
+                            starting_affine=translation.affine)
+
+    # affine transform (translation + rotation + scaling)
+    affine = affreg.optimize(img_s, img_m, AffineTransform3D(), None,
+                             img_s_grid2world, img_m_grid2world,
+                             starting_affine=rigid.affine)
+
+    # apply affine transformation
+    img_m_affine = affine.transform(img_m)
+
+    # set up Symmetric Diffeomorphic Registration (metric, iterations)
+    sdr = SymmetricDiffeomorphicRegistration(CCMetric(3), list(niter_sdr))
+
+    # compute mapping
+    mapping = sdr.optimize(img_s, img_m_affine)
+
+    return mapping, affine
+
+
+###############################################################################
+# Apply non-linear morph mapping
+
+def morph_precomputed(img, affine, mapping):
+    # morph img data
+    img_sdr_affine = np.zeros(img.shape)
+    for vol in range(img.shape[3]):
+        img_sdr_affine[:, :, :, vol] = mapping.transform(
+            affine.transform(img.dataobj[:, :, :, vol]))
+
+    return img_sdr_affine
+
+
+###############################################################################
+# Execute example
+
+# Setup path
+data_path = sample.data_path()
+
+# compute LCMV beamformer inverse example
+compute_lcmv_example_data(data_path)
+
+# voxel size for reslicing
+voxel_size = (3., 3., 3.)  # consider changing for real data scenario
+
+# load lcmv inverse
+img_vol = nib.load('lcmv_inverse.nii.gz')
```

Comment:
maybe you can avoid saving to disk here and make compute_lcmv_example_data return an img